docs/_tutorials/getting_started.md

```diff
@@ -104,63 +106,60 @@ const app = new App({
 })();
 ```
 
-Your token and signing secret are enough to create your first Bolt app. Save your `app.js` file then on the command line run the following:
+Save your `app.js` file, then on the command line run the following:
 
 ```script
 node app.js
 ```
 
-Your app should let you know that it's up and running.
+Your app should let you know that it's up and running. 🎉 
 
 ---
 
 ### Setting up events
-Your app behaves similarly to people on your team — it can post messages, add emoji reactions, and more. To listen for events happening in a Slack workspace (like when a message is posted or when a reaction is posted to a message) you'll use the [Events API to subscribe to event types](https://api.slack.com/events-api).
+Your app behaves similarly to people on your team — it can post messages, add emoji reactions, and listen and respond to events. 
 
-To enable events for your app, start by going back to your app configuration page (click on the app [from your app management page](https://api.slack.com/apps)). Click **Event Subscriptions** on the left sidebar. Toggle the switch labeled **Enable Events**. 
+To listen for events happening in a Slack workspace (like when a message is posted or when a reaction is posted to a message) you'll use the [Events API to subscribe to event types](https://api.slack.com/events-api). For this guide, we are going to be using [Socket Mode](https://api.slack.com/apis/connections/socket), our recommended option for those just getting started and building something for their team. 
 
-You'll see a text input labeled **Request URL**. The Request URL is a public URL where Slack will send HTTP POST requests corresponding to events you specify.
+> 💡 Socket Mode allows your app to use the Events API and interactive components of the platform—without exposing a public HTTP Request URL. This mode is helpful for  development, or if you're receiving requests from behind a firewall. HTTP is more useful for apps being deployed to a hosting environment, or apps intended for distribution via the Slack App Directory. You can easily switch between Socket Mode and a direct HTTP endpoint and at any time in the app config. 
```

Comment:
```suggestion
> 💡 Socket Mode allows apps to use the Events API and interactive components without exposing a public HTTP endpoint. This can be helpful during development, or if you're receiving requests from behind a firewall. HTTP is more useful for apps being deployed to hosting environments (like [AWS](/bolt-js/deployments/aws-lambda) or [Heroku](/bolt-js/deployments/heroku)), or apps intended for distribution. You can easily toggle between Socket Mode and HTTP connections at any time in the app config. 
```

Reply:
I see you took out the hosting providers line below—while I think it may be helpful to link to the AWS and Heroku guides in this, feel free to leave it out if you think it adds clutter or disagree for another reason.